core/arch/x86/decode_table.c

```diff
@@ -1483,6 +1483,19 @@ const instr_info_t * const op_instr[] =
 #define KEw TYPE_K_EVEX, OPSZ_2
 #define KEd TYPE_K_EVEX, OPSZ_4
 #define KEq TYPE_K_EVEX, OPSZ_8
+#define KEVps TYPE_K_EVEX_V, OPSZ_16_32_evex64
```

Comment:
What's the rationale behind this? There shouldn't be any ps/pd in the operand type + size combo. The ps/pd relates to element size and not to operand size (e.g. the 64-byte vector is still a 64-byte vector, no matter whether ps/pd.

Reply:
Ok. I will change with OPSZ_64.

Reply:
> There shouldn't be any ps/pd in the operand type + size combo.
Clarification: w.r.t. mask register. Actually TBD, not sure right now. but..

.. the KE* types are the mask operand types. Specifically the EVEX.aaa mask register, which is 8 bytes max. (There is no OPSZ_64 opmask.)

Reply:
@hgreving2304 I understood you. With changing size of operand  I going on to change type name too. In above comment I described why type was named it this way.